tests/search.test.js

```diff
@@ -0,0 +1,56 @@
+import chai from 'chai';
+import '@babel/polyfill';
+import chaiHttp from 'chai-http';
+import status from '../src/helpers/constants/status.codes';
+import errors from '../src/helpers/constants/error.messages';
+
+import server from '../src/index';
+
+chai.use(chaiHttp);
+chai.should();
+
+describe('Search functinality', () => {
+  it('Should search for avalaible articles', (done) => {
+    chai
+      .request(server)
+      .get('/api/articles/search?title=What')
+      .end((err, res) => {
+        res.should.have.status(status.OK);
+        res.body.should.have.property('articles');
```

Comment:
@alainburindi we've talked about checking for the exact content of the result returned, and not just the properties of the response object. With this test, how do we know that our search returns articles containing the search key, 'What'